cmd/mtbin.cpp

```diff
@@ -14,48 +14,358 @@
 
 #include "command.h"
```

Comment:
Just a gentle warning about this commit, so it doesn't go unnoticed.  I personally highly discourage the use of this mtbin code.